extensions/ql-vscode/src/remote-queries/variant-analysis-view.ts

```diff
@@ -103,152 +93,20 @@ export class VariantAnalysisView extends AbstractWebview<ToVariantAnalysisMessag
-            fullName: 'octodemo/hello-world-4',
-            private: false,
-          },
-          analysisStatus: VariantAnalysisRepoStatus.Pending,
-        },
-        {
-          repository: {
-            id: 5,
-            fullName: 'octodemo/hello-world-5',
-            private: false,
-          },
-          analysisStatus: VariantAnalysisRepoStatus.Pending,
-        },
-        {
-          repository: {
-            id: 6,
-            fullName: 'octodemo/hello-world-6',
-            private: false,
-          },
-          analysisStatus: VariantAnalysisRepoStatus.Pending,
-        },
-        {
-          repository: {
-            id: 7,
-            fullName: 'octodemo/hello-world-7',
-            private: false,
-          },
-          analysisStatus: VariantAnalysisRepoStatus.Pending,
-        },
-        {
-          repository: {
-            id: 8,
-            fullName: 'octodemo/hello-world-8',
-            private: false,
-          },
-          analysisStatus: VariantAnalysisRepoStatus.Pending,
-        },
-        {
-          repository: {
-            id: 9,
-            fullName: 'octodemo/hello-world-9',
-            private: false,
-          },
-          analysisStatus: VariantAnalysisRepoStatus.Pending,
-        },
-        {
-          repository: {
-            id: 10,
-            fullName: 'octodemo/hello-world-10',
-            private: false,
-          },
-          analysisStatus: VariantAnalysisRepoStatus.Pending,
-        },
-      ],
-      skippedRepos: {
-        notFoundRepos: {
-          repositoryCount: 2,
-          repositories: [
-            {
-              fullName: 'octodemo/hello-globe'
-            },
-            {
-              fullName: 'octodemo/hello-planet'
-            }
-          ]
-        },
-        noCodeqlDbRepos: {
-          repositoryCount: 4,
-          repositories: [
-            {
-              id: 100,
-              fullName: 'octodemo/no-db-1'
-            },
-            {
-              id: 101,
-              fullName: 'octodemo/no-db-2'
-            },
-            {
-              id: 102,
-              fullName: 'octodemo/no-db-3'
-            },
-            {
-              id: 103,
-              fullName: 'octodemo/no-db-4'
-            }
-          ]
-        },
-        overLimitRepos: {
-          repositoryCount: 1,
-          repositories: [
-            {
-              id: 201,
-              fullName: 'octodemo/over-limit-1'
-            }
-          ]
-        },
-        accessMismatchRepos: {
-          repositoryCount: 1,
-          repositories: [
-            {
-              id: 205,
-              fullName: 'octodemo/private'
-            }
-          ]
-        }
-      },
-    };
+  protected async onWebViewLoaded() {
+    super.onWebViewLoaded();
+
+    void logger.log('Variant analysis view loaded');
+
+    const variantAnalysis = await this.manager.getVariantAnalysis(this.variantAnalysisId);
+
+    if (!variantAnalysis) {
```

Comment:
Should we show a notification to the user to tell them something went wrong?

Reply:
I think that would indeed be useful, I'll add a `showAndLogWarningMessage` call here.